.github/workflows/build.yml

```diff
@@ -367,11 +386,62 @@ jobs:
         run: make package/linux/compile -j$(nproc) BUILD_LOG=1 || ret=$? .github/workflows/scripts/show_build_failures.sh
 
       - name: Build everything
-        if: inputs.build_full == true
+        if: inputs.build_full == true || inputs.coverity_check_packages != ''
```

Comment:
Why not provide the `build_full` variable in coverity builds?

Reply:
Good catch, thanks.